src/main/java/mate/academy/MergeSortAction.java

```diff
@@ -1,8 +1,54 @@
 package mate.academy;
 
-public class MergeSortAction {
+import java.util.Arrays;
+import java.util.concurrent.RecursiveAction;
+
+public class MergeSortAction extends RecursiveAction {
+    private static final int THRESHOLD = 1;
+    private final int[] array;
 
     public MergeSortAction(int[] array) {
+        this.array = array;
+    }
+
+    @Override
+    protected void compute() {
+        int start = 0;
```

Comment:
The variable 'start' is initialized but never used other than in the copyOfRange method where it is always 0. If the intention is to sort the entire array, the variable is unnecessary. If you plan to extend functionality to sort subarrays, then 'start' needs to be a parameter of the MergeSortAction constructor.

Reply:
On my opinion, if I write '0' without a variable, it will be considered a magical number. Therefore, I created a local variable to prevent this.